### DISCUSSION TITLE
让沉浸式翻译不仅仅是翻译工具！

### DISCUSSION THREAD

**Discussion by BruceAi66** [immersive-translate/immersive-translate#1332] [Ideas]
### 新功能 | New Feature

1.划词查词。这样我就可以卸载调其它的查词插件了（沙拉查词）。
2.保存单词或句子。参考Language Reactor。
3.支持导出保存的项目到ANKI ，参考Language Reactor。

### 为什么您想要添加这些功能 | Why you want to add these features

1.如果添加划词查词的功能需要的工作量不大，何乐而不为呢，否则，请开发者酌情考虑。
2.有了保存单词或语句的功能，可以让翻译插件升华到不仅是翻译，更是学习的辅助插件。
3.支持导出保存项目到ANKI，主要是进一步促进学习的效果。

### 提议 | Proposal

可以参考Language Reactor  ，它主打语言学习，功能设计不错，但是翻译质量就是谷歌免费翻译的水平，又不能像沉浸式一样可以自定义翻译源，而鉴于它方便的保存和导出到ANKI的功能（收费的附加功能），目前我仍然主要是用Language Reactor来翻译视频字幕 。  而在文章全文翻译（双语）方面，沉浸式翻译是无与伦比的，Language Reactor的处理方式非常蹩脚，我们可以参考下它的保存和导出的功能设计，其它的方便沉浸式翻译都堪称完美。

**Comment by theowenyoung** [comment]
主要是我们目前的观点是不希望沉浸式翻译变得更加复杂和全面，只做好一件事。 所以你可能还是要配合不同的工具实现自己的需求。

**Comment by BruceAi66** [comment]
好吧，保持简洁也是我非常认可的和支持的。

Owen ***@***.***> 于2024年1月17日周三 19:30写道：

**Comment by KnIfER** [comment]
划词已在“抹布翻译”脚本中实现：#1916
